src/rollup/validation/validate-widget-position.js

```diff
@@ -13,7 +13,7 @@ export default function validateWidgetPosition (widget, config) {
 
   switch (config.layout) {
   case 'modal':
-    choices = [''];
+    choices = ['', 'middle-center'];
```

Comment:
Why are you leaving empty string here?
I_believe_ modal/middle-center is a default from the initial creation of the model (the pre-selections that people can change), but they can't change it to nothing.

Reply:
Also, maybe I don't understand what's going on here, but we seem to be 
- missing cases:
`sticky-bar` (top-center, bottom-center)
`gate` (also middle-center only)
- including a case I don't recognize (button?)
- referring to the choices differently (bar is now just top-center; what's "top-absolute"?)

Reply:
@rallyb13 The pathfora position scheme was different than the ones set up for the API. In pathfora modals don't have a position as there is only one. I have now included the missing case for  stickybar. And in Pathfora gate is considered a modal. So we should be good.